code/modules/clothing/gloves/miscellaneous.dm

```diff
@@ -166,7 +166,8 @@
 	var/mob/living/M = loc
 
 	if(M.a_intent in accepted_intents)
-		if(istype(M.mind.martial_art, /datum/martial_art/cqc))
+		if(istype(M.mind.martial_art, /datum/martial_art))
+		if(HAS_TRAIT(user, HULK_TRAIT))
 			M.changeNext_move(CLICK_CD_MELEE)//normal attack speed for hulk, CQC and Carp.
```

Comment:
You probably want this instead:
```suggestion
		if(istype(M.mind.martial_art, /datum/martial_art) || HAS_TRAIT(M, TRAIT_HULK))
			M.changeNext_move(CLICK_CD_MELEE)//normal attack speed for hulk, CQC and Carp.
```
Or to make that a bit more readable:
```dm
		if((M.mind has any martial arts) OR (M has the Hulk trait))
				Change the click delay back to normal.
		else
				Change the click delay to the fast version.
```

Reply:
I also suggest doing `if(M.mind.martial_art)` instead of the istype, because, the only thing that's ever assigned to that to that var is a `/datum/martial_art`.

Reply:
Right, followed your suggestions. Thank you all for being so incredibly helpful. As you could most likely tell this is my first PR and all of this is moon runes to my primitive rodent brain.